mptt/admin.py

```diff
@@ -113,6 +115,7 @@ class DraggableMPTTAdmin(MPTTModelAdmin):
     list_display_links = ('indented_title',)  # Sane defaults.
     mptt_level_indent = 20
     expand_tree_by_default = False
+    log_move_node_action = True
```

Comment:
Is there a reason for making this optional?

Reply:
Backward compatibility only. If you think it's not necessary, I'll remove it. let me know

Reply:
My opinion is that it's better to avoid branches if possible -- if someone later objects they can still introduce the conditional.